controllers/metal3data_controller_test.go

```diff
@@ -118,17 +125,14 @@ var _ = Describe("Metal3Data manager", func() {
 			Entry("Metal3Data not found", testCaseReconcile{}),
 			Entry("Missing cluster label", testCaseReconcile{
 				m3d: &infrav1.Metal3Data{
-					ObjectMeta: metav1.ObjectMeta{
-						Name:      "abc",
-						Namespace: "def",
-					},
+					ObjectMeta: testObjectMeta,
 				},
 			}),
 			Entry("Cluster not found", testCaseReconcile{
 				m3d: &infrav1.Metal3Data{
 					ObjectMeta: metav1.ObjectMeta{
 						Name:      "abc",
-						Namespace: "def",
+						Namespace: "myns",
 						Labels: map[string]string{
 							capi.ClusterLabelName: "abc",
 						},
```

Comment:
these could be defined like 'ObjectMeta: testObjectMetaWithLabel' to avoid repetition

Reply:
done, thanks